.changeset/six-boats-yell.md

```diff
@@ -0,0 +1,10 @@
+---
+"wc-next13": patch
```

Comment:
I don't think integrating PIE Button would be a patch. I think it would be minor.